tests/basic/misc_test.go

```diff
@@ -105,6 +165,22 @@ var _ = Describe("{VolumeDriverDown}", func() {
 					}
 				})
 			}
+
+			if Inst().V.String() == portworx.PureDriverName {
+				// Finish up the write routines from earlier
+				// First, check for any errors
+				var err error
+				select {
+				case err = <-pureErrOutChan:
+					log.FailOnError(err, "Error writing to Pure volume")
+				default:
+					log.Infof("No errors found in error channel for Pure volume write validation")
+				}
+				// Then, close all the routines out so they stop writing
+				pureStopWriteRoutine = true
+				return
```

Comment:
Do we not need line 184 in case of FADA?

Reply:
We do not - that runs tests that are specific to pxd volumes (it runs commands like pxctl sv md5sum or something like that?)